meta-xt-driver-domain/recipes-graphics/libsdl2/libsdl2_2.0.20.bbappend

```diff
@@ -0,0 +1,2 @@
+
+PACKAGECONFIG:append = "${@' kmsdrm gles2 ' if 'wayland' not in d.getVar('DISTRO_FEATURES', True).split() else ''}"
```

Comment:
This is very convoluted way to write ` ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', '', 'kmsdrm gles2 ', d)}` :)

Please rewrite it in Yocto-way

Reply:
Done